content/docs/start/data-and-model-versioning.md

```diff
@@ -109,10 +109,9 @@ $ git commit -m "Configure remote storage"
 
 DVC remotes let you store a copy of the data tracked by DVC outside of the local
 cache (usually a cloud storage service). For simplicity, let's set up a _local
-remote_:
+remote_ in `/tmp/dvcstore` (create the dir first if needed):
 
 ```dvc
-$ mkdir -p /tmp/dvcstore
 $ dvc remote add -d myremote /tmp/dvcstore
```

Comment:
OK so here's another issue with trying to be Windows-friendly @shcheklein : There's no root dit `/`. So we'd need to change this whole example in every command even simple `tree` ones to use some other path to a temporary folder, I guess `%TEMP%` or `%userprofile%\AppData\Local\Temp` (not sure, requires Windows knowledge and I wonder if it's important).

Reply:
There is no `/tmp` on macOS either.

For Linux/macOS `/tmp` can be changed to `$TMPDIR`. It's fairly standard environment variable. In Windows, that seems to be `%temp%`, so the above command should read:

```powershell
mkdir -Path %TEMP%\dvcstore
```

Reply:
PS `mkdir` also uses `-p` for `-Path`, not for `--parent` as in Bash.

Reply:
The point is that I'm not sure how much effort we want to put into making things Windows friendly e.g. in cases where we need tabs just because the system paths don't make sense in Windows, or even potentially just to change `/` for `\`.

Reply:
I believe Windows commands could be used directly like in Bash. If we're giving examples, they can rather be correct to run as is.

Reply:
In macOS, some of my scripts don't run because `vim /tmp/${RANDOM}.txt` gives an error, I have to replace those lines. It may be a permissions error but `/tmp` is not as straightforward to use, and `$TMPDIR` points to elsewhere under  `$HOME`.

Reply:
`/tmp/${RANDOM}.txt` works fine on macOS for me, same as `/$TEMP/${RANDOM}`.

TBH I'm not sure what you're proposing @iesahin 🙂  - should we review all the code samples to make sure they work on Windows? Including valid commands, paths, `\` dir. separators, etc.

Reply:
The true user-isolated way to use temp directory in macOS is by `$TMPDIR`, but that environment variable is not available in all Linux distributions, so we'll have to create three tabs, one for Linux, one for macOS and one for Windows, which looks like an overkill to  me. 

Instead, let's keep Linux/macOS with `/tmp/...` and use `%TEMP%` in Windows, and check all commands as you suggest.

Reply:
OK. Also as discussed in our last retro it's worth it going all the way with complete/consistent Windows tabs throughout Get Started pages so I'll do so here. Not super convinced that commands in hidden sections need that too but oh well, it doesn't hurt...